view/frontend/templates/hyva-topmenu-desktop/menu/sub_menu.phtml

```diff
@@ -14,16 +14,13 @@ $heroicons = $viewModels->require(\Hyva\Theme\ViewModel\HeroiconsOutline::class)
     <?php if ($level === 1) : ?>
         <div
             x-ref="submenu-<?= /* @noEscape */ (string) $parentNodeIndex ?>"
-            class="absolute left-0 top-full -mt-4 w-full z-10 shadow-lg bg-white"
-            :class="{
-                'hidden' : openSubmenuId !== '<?= /* @noEscape */ (string) $parentNodeIndex ?>',
-                'flex' : openSubmenuId === '<?= /* @noEscape */ (string) $parentNodeIndex ?>',
-            }"
+            class="absolute left-0 top-full w-full z-30 shadow-md bg-white flex"
+            x-show="openSubmenuId === '<?= /* @noEscape */ (string) $parentNodeIndex ?>'"
```

Comment:
Not strictly related with this PR but please fix issue with lazy loading for images inside the mega menu. Perhaps it doesn't work because of x-clock.  Please check but IMO we can get rod of the x-clock and add hdiden class as a default.